modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,135 @@
++
+----
+$ curl -J -L -O https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-vmware.ova
+----
+
+. In the vSphere Client, create a folder in your datacenter to store your VMs.
+The folder name must match the cluster name that you specified in the
+`install-config.yaml` file.
+
+. In the vSphere Client, create a template for the OVA image.
++
+[NOTE]
+====
+In the following steps, you use the same template for all of your cluster
+machines and provide the location for the Ignition config file for that machine
+type when you provision the VMs.
+====
+.. From the *Hosts and Clusters* tab, right-click your cluster's name and
+click *Deploy OVF Template*.
+.. On the *Select an OVF* tab, specify the name of the {op-system} OVA file
+that you downloaded.
+.. On the *Select a name and folder* tab, set a *Virtual machine name*, such
+as {op-system}, and click the name of the your vSphere cluster.
+.. On the *Select a computer resource* tab, click the name of the your vSphere
+cluster.
+.. On the *Select storage* tab, configure the storage options for your VM.
+For example, select *Thin Provision* if you plan to supply
+additional storage to the cluster instead of using storage on the machines, and
+select the datastore that you specified in your `install-config.yaml` file.
+.. On the *Select network* tab, specify the the network that you configured
+for the cluster, if available.
+.. If you plan to use the same template for all cluster machine types, do not
+specify values on the *Customize template* tab.
+
+. After the template deploys, deploy a VM for a machine in the cluster.
+.. Right-click the template's name and click *Clone* -> *Clone to Virtual Machine*.
+.. On the *Select a name and folder* tab, specify a name for the VM. You might
+include the machine type in the name, such as `control-plane-0` or `compute-1`.
+.. On the *Select a name and folder* tab, select the name of the folder that
+you created for the cluster.
+.. On the *Select a compute resource* tab, select the name of a host in your
+datacenter.
++
+[IMPORTANT]
+====
+You must specify the same host for each machine in your cluster.
+====
+.. Optionally, on the *Select storage* tab, customize the storage options.
+.. On the *Select clone options*, select
+*Customize this virtual machine's hardware*.
+.. On the *Customize hardware* tab, click *VM Options* -> *Advanced* -> *Edit Configuration*.
+.. On the *Configuration Parameters* window, click *Add Configuration Params*
+and define the following parameter names and values:
+*** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded
+Ignition config file for this machine type.
+*** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
+*** `enable_disk_uuid`: Specify `true`.
+.. Review the contents of the *Virtual Hardware* panel of the
+*Customize hardware* tab, modify the specified values as required. Ensure that
```

Comment:
Should we list a best practice guide for creating the VMs? There are options like setting eager thick disk and high latency sensitivity that will benefit performance greatly.

Reply:
Can you tell me more about the requirements? @staebler said that it might be better to use thin disks to decrease space utilization, but if another way is better, I'm happy to change it.

Reply:
My recommendation for thin provision comes from https://coreos.com/os/docs/latest/booting-on-vmware.html. It may be outdated.

Reply:
It should be fine. There were a few issues where RH and VMware support collaborated on performance issues. The solution was setting high latency on VMs used in the OCP cluster. 

Here is the option in terraform:
https://www.terraform.io/docs/providers/vsphere/r/virtual_machine.html#latency_sensitivity

This could also be manually tuned as per: 
https://access.redhat.com/documentation/en-us/reference_architectures/2018/html-single/deploying_and_managing_openshift_3.9_on_vmware_vsphere/index#prior_to_ansible_installation

Reply:
I'm adding the step to set high latency. Please let me know if you want me to remove it.